pkgs/tools/misc/zellij/default.nix

```diff
@@ -22,13 +22,17 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
-    installShellCompletion assets/completions/zellij.{bash,fish} --zsh assets/completions/_zellij
+    mkdir -p assets/completions
+    for shell in fish bash zsh; do
+      $out/bin/zellij generate-completion $shell > assets/completions/zellij.$shell
+    done
+    installShellCompletion assets/completions/zellij.{bash,fish} --zsh assets/completions/zellij.zsh
```

Comment:
```suggestion
    installShellCompletion --cmd \
      --bash <($out/bin/zellij generate-completion bash) \
      --fish <($out/bin/zellij generate-completion fish) \
      --zsh assets/completions/zellij.zsh
```

Reply:
I think this one needs a little tweak, since the assets/completions/zellij.zsh needs to be created. But the idea of trying to do it in a single command is good I think.